src/query/ast/src/parser/statement.rs

```diff
@@ -1970,6 +1970,52 @@ pub fn alter_database_action(i: Input) -> IResult<AlterDatabaseAction> {
     )(i)
 }
 
+fn column_type(i: Input) -> IResult<(Identifier, TypeName)> {
+    map(
+        rule! {
+            #ident ~ #type_name
+        },
+        |(column, type_name)| (column, type_name),
+    )(i)
+}
+
+pub fn modify_column_action(i: Input) -> IResult<ModifyColumnAction> {
+    let set_mask_policy = map(
+        rule! {
+            #ident ~ SET ~ MASKING ~ POLICY ~ #ident
+        },
+        |(column, _, _, _, mask_name)| {
+            ModifyColumnAction::SetMaskingPolicy(column, mask_name.to_string())
+        },
+    );
+
+    let convert_stored_computed_column = map(
+        rule! {
+            #ident ~ DROP ~ STORED
+        },
+        |(column, _, _)| ModifyColumnAction::ConvertStoredComputedColumn(column),
+    );
+
+    let modify_column_type = map(
+        rule! {
+            DATA ~ TYPE ~ #column_type ~ ("," ~ #column_type)*
```

Comment:
This `DATA ~ TYPE` keyword is no need, it looks special.

`ALTER TABLE <name> MODIFY  [<col1_name> <type>][,<col1_name> <type>]`